setup/scripts/install-brew.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo "Installing Homebrew..."
+
+# Check if Homebrew is already installed
+if command -v brew &>/dev/null; then
+    echo "Homebrew is already installed!"
+else
+    echo "Homebrew not found. Installing Homebrew..."
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+fi
+
+# Check if Homebrew is working
+if ! command -v brew &>/dev/null; then
+    echo "Error: Homebrew installation failed!"
+    exit 1
+fi
+
+echo "Homebrew installation complete!"
+brew update && brew upgrade
```

Comment:
minor: missing newline end of file which I believe is standard lint practice. We should probably add .sh into formatting in workspace.